feature-utils/poly-import/src/entities/data-account.js

```diff
@@ -1,14 +1,31 @@
+import { runImporter } from "../importer";
+
 export default class DataAccount {
     constructor() {
-        this.importingResults = [];
+        this.importingReports = [];
         this.importedFileNames = [];
-        this.personalData = { name: { givenName: "", familyName: "" } };
-        this.preferredLanguage = [];
         this.analyses = {};
         this.reports = {};
         this.analysesExecutionResults = [];
     }
 
+    async import({ importers, zipFile, pod }) {
+        for (let [attr, importerClass] of Object.entries(importers)) {
```

Comment:
You can probably just do

```javascript
const attr, importerClass = Object.entries(importers)
```

without using the loop construct. It's just destructuring you're looking for, right?